src/components/FilmItem/index.js

```diff
@@ -1,16 +1,60 @@
-import React from 'react';
+import React, { useState, useRef, useEffect } from 'react';
 import PropTypes from 'prop-types';
 
 import "./FilmItem.sass"
 
 import filmCardPlaceholder from './../../assets/images/placeholder_320x455_ffffff_cccccc.jpg'
 
-const FilmItem = ({ title, genre, release_date, thumbnail }) => {
+const FilmItem = ({ title, genre, release_date, thumbnail, onClickEdit, onClickDelete }) => {
     const checkFilmCardThumbnail = thumbnail ?
         thumbnail : filmCardPlaceholder;
+
+    const [showMenu, setShowMenu] = useState(false)
+    const menuRef = useRef(null)
+    const handleClickOutside = (event) => {
+        if (menuRef.current && !menuRef.current.contains(event.target)) {
+            setShowMenu(false)
+        }
+    }
+
+    useEffect(() => {
+        document.addEventListener('click', handleClickOutside, true)
+        return () => {
+            document.removeEventListener('click', handleClickOutside, true)
+        }
+    })

```

Comment:
```suggestion
    }, [handleClickOutside])
```
You could read more about `useEffect` dependencies [here](https://ru.reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect).